src/service/platform-impl.js

```diff
@@ -158,6 +158,20 @@ export class Platform {
     version = version[1].replace(/_/g, '.');
     return version;
   }
+
+  /**
+   * Check whether current major IOS version is after specific version number.
+   * @param {number} version
+   * @return (boolean}
+   */
+  afterIosMajorVersion(version) {
```

Comment:
Later I want to compare ios minor version and have a method `afterIosMinorVersion('10.3.1')`. If we want to use `getIosMinorVersion`, then when comparing we could not use `<`/`>` directly, instead we should use something like:
```
const iosMinorVersion = getIosMinorVersion();
if (isVersionNumberAfter(iosMinorVersion, '10.3.1')) {
  ...
}
```
In the above code, we have to use two methods instead of one. However, I prefer having only one method `afterIosMinorVersion`. If we have `afterIosMinorVersion`, then I think it's better to use `afterIosMajorVersion`.